src/main/webapp/i18n/de/course.json

```
@@ -209,6 +209,13 @@
                 "lowest": "Unteres Performance-Drittel",
                 "average": "Durchschnittliches Performance-Drittel",
                 "best": "Bestes Performance-Drittel"
+            },
+            "calendarWeek": "Kalenderwoche",
+            "scopeButton": {
+                "recentWeekButton": "Jüngste Aktivität",
+                "recentWeekTooltip": "Zeigt die Aktivität von 17 Wochen. Begann der Kurs vor weniger als 17 Wochen, beginnt die Anzeige zum Startdatum.",
+                "lifetimeOverviewButtonLabel": "Kursdauerüberblick",
+                "lifetimeOverviewTooltip": "Zeigt die Periode von Kurstart bis derzeitigem Datum. Ist der Kurs bereits beendet, endet die Anzeige zum Enddatum des Kurses."
```

```suggestion
                "lifetimeOverviewTooltip": "Zeigt die Periode von Kurstart bis jetzt. Ist der Kurs bereits beendet, endet die Anzeige zum Enddatum des Kurses."
```
Personal preference: can be shortened without losing information.